src/pages/plugins.tsx

```diff
@@ -21,6 +22,10 @@ export const PluginLibraryPage = () => {
 
     return (
         <Layout>
+            <SEO
+                title="Plugin Library • PostHog"
+                description="Plugins for getting data in and out of PostHog, the open source product analytics platform."
```

Comment:
That's fine for now! We'll do another pass once we've made a plugins marketing page -  I've already written some marketing for it on my SuperDay.